MetaMask/metamask-extension
# Bump test-dapp to v7.1.0

## Explanation
Bump test-dapp to v7.1.0
See changes [here](https://github.com/MetaMask/test-dapp/pull/254).

* Fixes https://github.com/MetaMask/MetaMask-planning/issues/1264

## Screenshots/Screencaps


https://github.com/MetaMask/metamask-extension/assets/54408225/0508d0e0-0b02-4f53-8a10-1f8ff3973856



### Before

`v7.0.1`

### After

`v7.1.0`

## Manual Testing Steps
- Check circle ci runs
- Locally run tests in your environment `yarn test:e2e:chrome`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
